textract/parsers/utils.py

```diff
@@ -31,7 +31,7 @@ def encode(self, text, encoding):
         """
         return text.encode(encoding, 'ignore')
 
-    def process(self, filename, encoding, **kwargs):
+    def process(self, filename, encoding, extension, **kwargs):
```

Comment:
It does not look like we need to pass `extension` to the `process` method. Please remove this here and in the call from `textract.process`